docs/BOOTING.md

```diff
@@ -136,6 +136,23 @@ in grub.cfg with graphical GRUB menu to get the device to boot again.
 3. General kernel parameters may be adjusted with `set_global dom0_extra_args "$dom0_extra_args OPTION1=VAL1 OPTION2 "`.
    They will be added to kernel cmdline.
 
+As an example of using serial port and VGA as a console the following kernel arguments
+should be used:
+
+```
+set_global dom0_extra_args "$dom0_extra_args console=ttyS0,115200n8 console=tty0"
+```
+
+defines that opening ``/dev/console`` will get you the current foreground virtual
```

Comment:
I think you meant these to be single-ticks.

Reply:
Done